Index: Backend/communication.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Backend/communication.py b/Backend/communication.py
--- a/Backend/communication.py	
+++ b/Backend/communication.py	
@@ -10,8 +10,10 @@
 }
 """
 User Table holds 
+User ID # it is auto incramented everytime a new user is created
 Username 
 User ID # it is auto incramented everytime a new user is created
+User password
 
 
 Notes Table holds 
